web/login.go

```diff
@@ -76,26 +82,30 @@ func LoginFunc(w http.ResponseWriter, r *http.Request) {
 	if data.Username == conf.Username && util.PasswordOK(conf.Password, data.Password) {
 		ld.ticker.Stop()
 		ld.failedTimes = 0
-		tokenInSystem = util.GenerateToken(data.Username)
 
 		// 设置cookie过期时间为1天
-		cookieTimeout := 24
+		cookieTimeout := time.Now().Add(time.Hour * time.Duration(24))
 		if conf.NotAllowWanAccess {
 			// 内网访问cookie过期时间为30天
-			cookieTimeout = 24 * 30
+			cookieTimeout = time.Now().Add(time.Hour * time.Duration(24*30))
 		}
 
+		// 生成token/expires
+		tokenInSystem.Token = util.GenerateToken(data.Username)
+		tokenInSystem.Expires = cookieTimeout
```

Comment:
```suggestion
		tokenInSystem.Expires = time.Now().AddDate(0, 0, cookieTimeout)
```

可以让 cookieTimeout 只表示天数（`1` 和 `30`），然后在生成 `Expires` 时使用 [`AddDate`](https://pkg.go.dev/time#Time.AddDate) 函数统一获取过期时间。